playbooks/install_stack.yaml

```diff
@@ -70,6 +70,29 @@
       content: "{{ role_data }}"
       mode: 0644
 
+  - name: Configure manila
+    when: manila_enabled
+    block:
+    - name: Add manila services
+      set_fact:
+        enabled_services: "{{ enabled_services + manila_services }}"
+      vars:
+        manila_services:
+        - /usr/share/openstack-tripleo-heat-templates/environments/ceph-ansible/ceph-mds.yaml
+        - /usr/share/openstack-tripleo-heat-templates/environments/manila-cephfsganesha-config.yaml
+
+    # This works round https://bugs.launchpad.net/tripleo/+bug/1911022
+    - name: Add manila extraconfig
+      set_fact:
+        standalone_extra_config: "{{ standalone_extra_config | combine(manila_extra_config) }}"
+      vars:
+        manila_extra_config:
+          ganesha_vip: "{{ public_api }}"
+
+    - name: Ensure ceph is enabled
+      set_fact:
+        ceph_enabled: true
```

Comment:
Ceph users who don't use loop devices won't be able to use Manila because they don't use that option (otherwise they get Ceph configured with a loop device).

The simplest option to me is to add a toggle to whether or not create the loop device, and use it where it's useful (in prepare_host when we create the loop device) and in standalone_parameters when configuring Ceph params.